ResearchKit/Common/ORKHealthAnswerFormat.m

```diff
@@ -284,9 +336,9 @@ - (ORKAnswerFormat *)impliedAnswerFormat {
         ORKNumericAnswerFormat *format = nil;
             HKUnit *unit = [self healthKitUserUnit];
         if (_numericAnswerStyle == ORKNumericAnswerStyleDecimal) {
-            format = [ORKNumericAnswerFormat decimalAnswerFormatWithUnit:[unit unitString]];
+            format = [ORKNumericAnswerFormat decimalAnswerFormatWithUnit:[unit localizedUnitString]];
```

Comment:
Thanks for asking! (I couldn't remember anymore so had to check) As it turns out, the way this works is that this is in the `impliedAnswerFormat` which is the answer format used to display the question to the user. The results are saved using `-resultWithIdentifier:answer:` which uses `unitString` instead.